src/views/Info/Info.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function Info() {
+  return <div></div>;
+}
+
+export default Info;
```

Comment:
Is this component a view? You need to show it in name. For example _ItfoView_, but Info is general meaning and you need to use more exact name.

Reply:
I wouldn't say it is a strict requirement to do so.

Reply:
Left the option without "View"